### ISSUE TITLE
Uni-HamGNN released weights: v1 `predictor.pkl` vs v2 `uni-hamgnn_2_1.pkl`, model hyperparameters, and training details for citation

### ISSUE THREAD

**Issue by hjunxi22** [QuantumLab-ZY/HamGNN#115]
We use the v1 weights `predictor.pkl` (Zenodo 15568557, 927,022,084 bytes), loaded with module aliases for `HamGNN_v_2_0` / `HamGNN_v_2_1`. The readme now points to the v2 weights `uni-hamgnn_2_1.pkl` (Zenodo 17239078). To cite and describe the model correctly, could you clarify:

1. Is v2 a re-trained model, or the same weights re-serialized for the v2.1 package layout? If re-trained, were the training set or protocol changed, and is there any accuracy difference on the 5,000-structure test set?
2. Which of the two corresponds to the results reported in the Nat. Mach. Intell. paper?
3. Could you share the model configuration of the released Uni-HamGNN (irreps, cutoff radius, number of layers, radial basis, total parameter count)? The non-SOC model on Zenodo 10827117 ships a `model_config.yaml`, but the Uni-HamGNN pickle has no accompanying config, so the hyperparameters can only be recovered by unpickling.
4. The validation set size, and whether the 5,000-structure test set is disjoint from the 44,000 training structures.
5. For completeness of the methods description, could you indicate the training hardware and approximate wall time (GPU model, number of GPUs, and days for stage 1 and stage 2)? Reviewers often ask for this for pretrained models used in a study.

Thank you.

**Comment by QuantumLab-ZY** [comment]
# Response to Questions

Thanks for the careful documentation, and apologies for the missing configuration file alongside the pickle. Answers to your questions are provided below.

1. **v1 vs. v2 weights**

   `uni-hamgnn_2_1.pkl` ([Zenodo 17239078](https://doi.org/10.5281/zenodo.17239078)) contains exactly the same weights as `predictor.pkl` ([Zenodo 15568557](https://doi.org/10.5281/zenodo.15568557)).

   The model was not retrained. The file was only re-serialized and adjusted for the v2.1 package layout, hence the module aliases for:

   ```text
   HamGNN_v_2_0
   HamGNN_v_2_1
   ```

   The training set and training protocol are unchanged. The predictions—and therefore the accuracy on the 5,000-structure test set—are identical between the two files.

2. **Correspondence with the paper**

   Because the two files contain identical weights and produce identical predictions, both correspond to the results reported in the *Nature Machine Intelligence* paper.

   Either record may be cited. However, we recommend citing the v2 record, [Zenodo 17239078](https://doi.org/10.5281/zenodo.17239078), which is currently linked in the README.

3. **Model configuration**

   The hyperparameters recovered by unpickling the checkpoint—including the irreducible representations, cutoff radius, number of layers, radial basis, parameter count, and related settings—are accurate and may be used directly in your Methods section.

   We acknowledge that the pickle file is distributed without an accompanying `model_config.yaml`. This is solely a packaging omission and does not indicate a different model configuration.

4. **Data splits**

   - **Validation set:** 5,000 structures
   - **Test set:** 5,000 structures
   - **Training set:** 44,000 structures

   The 5,000-structure test set is disjoint from the 44,000 training structures.

5. Training hardware and wall time: the model was trained incrementally over multiple sessions rather than in a single continuous run, so we are unable to provide an accurate total wall-time figure, and we would rather not give a misleading estimate.
